api/src/main/java/org/openmrs/module/interop/api/observers/encounter/EncounterCreationObserver.java

```diff
@@ -33,6 +39,21 @@ public class EncounterCreationObserver extends BaseObserver implements Subscriba
 	@Qualifier("fhirR4")
 	private FhirContext fhirContext;
 	
+	@Autowired
+	private EncounterTranslator<Encounter> encounterTranslator;
+	
+	@Autowired
```

Comment:
Can we collapse those @Autowired annotations into one with constructor-injection